packages/static-build/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "node build",
     "test": "jest --env node --verbose --bail --runInBand",
-    "test-unit": "pnpm test test/build.test.ts test/prepare-cache.test.ts",
+    "test-unit": "pnpm test test/build.test.ts test/gatsby.test.js test/prepare-cache.test.ts",
```

Comment:
I just noticed there is a `builds.test.js` that is being ignored. Do we know why?

Reply:
I mentioned that in the PR description. No idea when or why its ignored, including the plethora of fixtures it references. I actually added a test to that file previously thinking it was being executed lol

regardless, the fixtures are out of date and so are the tests which is why this is all net-new